ql/src/test/queries/clientpositive/load_drop_partition.q

```diff
@@ -0,0 +1,24 @@
+dfs -cp ${system:hive.root}/data/files/table.txt ${system:test.tmp.dir}/;
+
```

Comment:
I think one qfile is enough. Let's test all the cases in `insert_overwrite_drop_partition.q`

Reply:
Added load overwrite partition case in one q file